src/brijesh/assignment22/PrimeNumbers.java

```diff
@@ -0,0 +1,22 @@
+package brijesh.assignment22;
+
+public class PrimeNumbers {
+	
+	void printPrimeNumbers(int start, int end) {
+		System.out.println("Prime numbers between "+start+" and "+end);
+		for (int index=start; index <= end; index++) {
+			boolean flag=true;
+			for(int innerIndex=2; innerIndex<=Math.ceil(Math.sqrt(index));innerIndex++) {
+				if(index%innerIndex==0)
+					flag=false;
```

Comment:
Here, we should give break, once we get that this number is not prime. 
Rest code very efficient